controller/env/sync.go

```diff
@@ -0,0 +1,73 @@
+/*
+	Copyright NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package env
+
+import (
+	"github.com/openziti/edge/controller/model"
+	"github.com/openziti/edge/controller/persistence"
+	"github.com/openziti/fabric/controller/network"
+)
+
+// Aliased type for router strategies
+type RouterSyncStrategyType string
+
+// Aliased type for router sync status
+type RouterSyncStatus string
+
+const (
+	RouterSyncNew        RouterSyncStatus = "SYNC_NEW"         //connection accepted but no strategy actions have been taken
```

Comment:
These are statuses that are usable by router sync strategies to report the status of an edge router. These are now output on the edge router list/detail endpoints.